tsconfig.json

```diff
@@ -24,5 +24,5 @@
     "strict": true,
     "target": "esnext"
   },
-  "include": ["src/**/*"]
+  "include": ["src/**/*", "addons/**/*"]
```

Comment:
remove this, since we already moved `addons` into `src`

Reply:
updated